packages/discord.js/src/index.js

```diff
@@ -97,6 +97,9 @@ exports.UnsafeEmbed = require('@discordjs/builders').UnsafeEmbed;
 exports.Emoji = require('./structures/Emoji').Emoji;
 exports.Guild = require('./structures/Guild').Guild;
 exports.GuildAuditLogs = require('./structures/GuildAuditLogs');
+/**
+ * @type {GuildAuditLogsEntry}
+ */
```

Comment:
Why?

Reply:
Because it can't compile without the explicit type annotation here.

Reply:
But it’s js ?

Reply:
Yea but it's run through the TS compiler.

Reply:
Alright ig